### PR TITLE
First cleanup package.json

### PR DESCRIPTION
**Pull request by martinheidegger**
The package json is not supposed to have development dependencies inside the regular dependencies. This PR cleans up the package json to include the author and only the development dependencies.

### CODE REVIEW

**Comment by m-schuetz** [comment]
Thanks for letting me know about that!

**Comment by m-schuetz** [comment]
Can you just redirect this PR to the develop branch?

**Comment by martinheidegger** [comment]
rebased

File: package.json [hunk 1]
```diff
@@ -3,17 +3,9 @@
...
-    "gulp-uglify": "*",
-    "gulp-rename": "*",
-    "jshint": "*",
-    "gulp-jshint": "*"
-  },
-  "author": {},
-  "license": "",
+  "dependencies": {},
+  "author": "Markus Schütz",
+  "license": "MIT",
```
Comment (m-schuetz) [inline package.json]: Please change license to "BSD-2-Clause"

**Comment by martinheidegger** [comment]
@m-schuetz done!

**Comment by m-schuetz** [comment]
Thanks!

**Pull request merged**